### DISCUSSION TITLE
vote: add spark-daria to mrpowers-io

### DISCUSSION THREAD

**Discussion by MrPowers** [mrpowers-io/community#10] [Ideas]
[spark-daria](https://github.com/MrPowers/spark-daria/) is a popular Spark helper library for Scala.

It needs to be updated and lightweight maintenance.

Proposed maintainers: Matthew Powers, Alf

**Comment by SemyonSinchenko** [comment]
+1 (binding)

**Comment by alfonsorr** [comment]
+1

**Comment by zeotuan** [comment]
+1

**Comment by MrPowers** [comment]
Vote passed.

Repo transferred: https://github.com/mrpowers-io/spark-daria

Project maintainers: me, @alfonsorr, and @zeotuan
